renku/cli/config.py

```diff
@@ -27,32 +27,79 @@
 
     $ renku config registry https://registry.gitlab.com/demo/demo
 
+By default, configuration is stored locally in the project's directory. Use
+``--global`` option to store configuration for all projects in your home
+directory.
+
+Remove values
+~~~~~~~~~~~~~
+
+To remove a specific key from configuration use:
+
+.. code-block:: console
+
+    $ renku config --remove registry
+
+By default, only local configuration is searched for removal. Use ``--global``
+option to remove a global configuration value.
+
 Query values
 ~~~~~~~~~~~~
 
-You display a previously set value with:
+You can display all configuration values with:
+
+.. code-block:: console
+
+    $ renku config
+
+Both local and global configuration files are read. Values in local
+configuration take precedence over global values. Use ``--local`` or
+``--global`` flag to read corresponding configuration only.
+
+You can provide a KEY to display only its value:
 
 .. code-block:: console
 
     $ renku config registry
     https://registry.gitlab.com/demo/demo
-
 """
 import click
 
-from renku.core.commands.config import update_config
+from renku.core import errors
+from renku.core.commands.config import read_config, update_config
 
 
 @click.command()
-@click.argument('key', required=True)
+@click.argument('key', required=False, default=None)
 @click.argument('value', required=False, default=None)
+@click.option('--remove', is_flag=True, help='Remove specified key.')
+@click.option(
+    '--local',
+    'local_only',
+    is_flag=True,
+    help='Read from local configuration only.'
```

Comment:
just a nitpick here, but perhaps this should use the same wording as below for the global option? i.e. `Read/store from/to local configuration only.`